src/SearchBar.scss

```diff
@@ -108,6 +117,11 @@
       padding-top: 3px;
     }
   }
+
+  .ais-VoiceSearch-status{
+    display: none;
```

Comment:
Is this used somewhere. `display: none` makes it permanently hidden.

Reply:
this status shows the info and status of the search, which is the default coming with the voicesearch widget. That's why we can only hide it.